gradle/lib/dependabot/gradle/file_parser.rb

```diff
@@ -59,6 +60,11 @@ def parse
         script_plugin_files.each do |plugin_file|
           dependency_set += buildfile_dependencies(plugin_file)
         end
+        if Experiments.enabled?(:gradle_wrapper_updater)
```

Comment:
@kbukum1 For my own understanding, what's the plan for this? How long do we normally run these as experiments and what % of the base users will see this running?